workflows/sars-cov-2-variant-calling/sars-cov-2-variation-reporting/variation-reporting.ga

```diff
@@ -45,7 +39,7 @@
             "uuid": "2e5a5b38-c204-45a2-98e2-e113bce5a14b",
             "workflow_outputs": [
                 {
-                    "label": null,
+                    "label": "af_filter_threshold",
```

Comment:
How did you create an output label here ? I don't think the editor offers this ?
This is where planemo fails. We should be able to give workflow output labels to optional inputs, but if you want the tests to pass for now you'll have to drop this.

Reply:
Fixed now in my WIP branch of planemo, that's why the tests have passed.